### PR TITLE
adding recipe for codeception 5.0

### PR DESCRIPTION
**Pull request by adrianmare**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     |

### CODE REVIEW

File: codeception/codeception/5.0/tests/Acceptance.suite.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Suite for acceptance tests.
+# Perform tests in browser using the WebDriver or PhpBrowser.
+# If you need both WebDriver and PHPBrowser tests - create a separate suite.
+
+actor: AcceptanceTester
+modules:
+    enabled:
+        - PhpBrowser:
+            url: http://localhost/myapp
+step_decorators: null
```
Comment (OskarStark) [inline codeception/codeception/5.0/tests/Acceptance.suite.yaml:12]: Indention looks wrong, it's not the same like in the next file in this PR

**Review by fabpot — CHANGES_REQUESTED** [review]
I would remove all PHP classes, which should not be part of a recipe.

**Comment by fabpot** [comment]
Duplicate work happening in #1432

**Comment by fabpot** [comment]
Closing in favor of #1432

**Pull request closed without merging**